app.rb

```diff
@@ -2,10 +2,14 @@
 
 require 'sinatra'
 require 'sinatra/reloader' if development?
-require 'workos'
+require 'dotenv/load' if development?
```

Comment:
Can we can remove the `if development?` bit? There aren't any .env files in the repo and it's sometimes desirable to run your development setup in different environments.